charts/rd-user-profile-api/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "0.1"
 description: Profile service for all users
 name: rd-user-profile-api
-version: 0.0.7
+version: 1.6.0
```

Comment:
why 1.6.0? should it be 0.0.8 if you want bump up?

Reply:
1.6.0 is what was proposed by Satya